account_invoice_rounding_by_currency/README.rst

```diff
@@ -0,0 +1,73 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================================================
+Unit rounded invoice (Swedish rounding) by currency
```

Comment:
I see that but it is NZ English ;)
I would suggest to switch to "Cash Rounding" which is more in line with American English used by Odoo.